app/Http/Controllers/User/UserController.php

```
@@ -0,0 +1,86 @@
+use App\Http\Controllers\Controller;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Gate;
+use App\Http\Requests\User\StoreUserRequest;
+use Illuminate\Support\Facades\Hash;
+use App\Http\Requests\User\UpdateUserRequest;
+use App\Http\Requests\User\ResetPasswordRequest;
+
+class UserController extends Controller
+{
+    public function index(Request $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)